redbot/cogs/audio/core/commands/llset.py

```diff
@@ -152,19 +152,19 @@ async def validate_input(cog, arg):
             ),
         )
 
-    @command_llset.command(name="external", aliases=["unmanaged"])
-    async def command_llset_external(self, ctx: commands.Context):
-        """Toggle using external Lavalink nodes - requires an existing external Lavalink node for Audio to work, if enabled.
+    @command_llset.command(name="unmanaged", aliases=["external"])
+    async def command_llset_unmanaged(self, ctx: commands.Context):
+        """Toggle using external (unmanaged) Lavalink nodes - requires an existing Lavalink node for Audio to work, if enabled.
 
-        This command disables the managed Lavalink server, if you do not have an external Lavalink node you will be unable to use Audio while this is enabled.
+        This command disables the managed Lavalink server. If you do not have another Lavalink node to access, you will be unable to use Audio while this is enabled.
```

Comment:
Same comment above, about being more clear that the user probably has to set something up if they use this setting.